docs/nodes/nodes-library/nodes/TimescaleDB/README.md

```diff
@@ -0,0 +1,97 @@
+
+### 2. HTTP Request node (GET)
+
+This node will make a GET request to the API `https://api.wheretheiss.at/v1/satellites/25544/positions` to fetch the position of the ISS. This information gets passed on to the next node in the workflow.
+::: v-pre
+1. Enter `https://api.wheretheiss.at/v1/satellites/25544/positions` in the ***URL*** field.
+2. Click on the ***Add Parameter*** button in the ***Query Parameters*** section.
+3. Enter `timestamps` in the ***Name*** field.
+4. Click on the gears icon next to the ***Value*** field and click on ***Add Expression***.
+5. Enter the following expression: `{{Date.now()}}`. This expression will return the current timestamp.
+6. Click on ***Execute Node*** to run the node.
+:::
+In the screenshot below, you will notice that the node makes a GET request to the API and returns the information about the location of the ISS.
+
+![Using the HTTP Request node to get the information about the location of the ISS](./HTTPRequest_node.png)
+
+### 3. Set node
+
+We will use the Set node to ensure that only the data that we set in this node gets passed on to the next nodes in the workflow.
+::: v-pre
+1. Click on ***Add Value*** and select 'String' from the dropdown list.
+2. Enter `latitude` in the ***Name*** field.
+3. Click on the gears icon next to the ***Value*** field and click on ***Add Expression***.
+4. Select the following in the ***Variable Selector*** section: Current Node > Input Data > JSON > 0 > latitude. You can also add the following expression: `{{$json["0"]["latitude"]}}`.
+5. Click on ***Add Value*** and select 'String' from the dropdown list.
+6. Enter `longitude` in the ***Name*** field.
+7. Click on the gears icon next to the ***Value*** field and click on ***Add Expression***.
+8. Select the following in the ***Variable Selector*** section: Current Node > Input Data > JSON > 0 > longitude. You can also add the following expression: `{{$json["0"]["longitude"]}}`.
+9. Click on ***Add Value*** and select 'String' from the dropdown list.
+10. Enter `timestamp` in the ***Name*** field.
+11. Click on the gears icon next to the ***Value*** field and click on ***Add Expression***.
+12. Select the following in the ***Variable Selector*** section: Current Node > Input Data > JSON > 0 > timpestamp. You can also add the following expression: `{{$json["0"]["timestamp"]}}`.
+13. Toggle ***Keep Only Set*** to `true`. We set this option to true to ensure that only the data that we have set in this node get passed on to the next nodes in the workflow.
+14. Click on ***Execute Node*** to run the node.
+:::
+In the screenshot below, you will notice that the node uses the data from the previous node and returns the data that we set for the workflow.
+
+![Using the Set node to set the data](./Set_node.png)
+
+### 4. TimescaleDB node (Insert)
+
+This node will push the data from the previous node to the `iss` table in TimescaleDB. If you have created a table with a different name, you can use that table instead.
```

Comment:
@harshil1712 Does TimescaleDB automatically populate the columns? If not, you'd need to mention exactly what kind of table they'd have to create and ideally provide a SQL statement that they can use to create that table.

Reply:
Made the change.